src/store/db.js

```diff
@@ -0,0 +1,49 @@
+import {fromJS, List} from 'immutable'
+
+// actions
+const INIT_DB = 'INIT_DB'
+
+const initDB = (db) => ({type: INIT_DB, db: db})
+
+// thunk
+
+export const fetchDb = dbName => dispatch => {
```

Comment:
Is it possible to write out what you mean here, rather than a double arrow function on 10? I don't easily follow the double arrow notation, and am sure I'll have more questions than anyone wants to sort it out, if I do. I'd appreciate being more verbose on all Thunks. If you're strongly opposed, I understand.

Reply:
for the thunk double arrow-  it’s exactly the same function as a normal thunk, its  just shorthand

so without the double arrow would be like this: 
function thunk (input) {
 return function (dispatch) {
   dispatch(input)
 }

the important thing is that it works exactly the same

Reply:
export const fetchDbNames = dbName ///this is the input to the outer function////  => dispatch /// this is the input to the inner function /// => {  /// now we are inside the inner / returned function as usual